### PR TITLE
Streamline build setup

### PR DESCRIPTION
**Pull request by hyc**
Use shorter, prefix-unique directory names
  gitian-builder  -> builder
  gitian-sigs     -> sigs
  monero-binaries -> out

Just use builder/inputs/monero, no need for 2nd repo clone

### CODE REVIEW

**Comment by dEBRUYNE-1** [comment]
@TheCharlatan - Perhaps you could review this PR?

File: contrib/gitian/gitian-build.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
 import subprocess
 import sys
 
+gsigs = 'https://github.com/monero-project/gitian.sigs.git'
+gbuild = 'https://github.com/devrandom/gitian-builder.git'
```
Comment (TheCharlatan) [inline contrib/gitian/gitian-build.py]: `gbuild` is probably a bit confusing, since there is another script with the same that is being run later on.

Comment (hyc) [reply contrib/gitian/gitian-build.py]: I renamed it to gbrepo.

File: contrib/gitian/gitian-build.py [hunk 2]
```diff
@@ -14,14 +17,21 @@ def setup():
...
-    os.chdir('gitian-builder')
+    if not os.path.isdir('sigs'):
+        subprocess.check_call(['git', 'clone', gsigs, 'sigs'])
+    if not os.path.isdir('builder'):
+        subprocess.check_call(['git', 'clone', gbrepo, 'builder'])
+    os.chdir('builder')
+    subprocess.check_call(['git', 'config', 'user.email', 'gitianuser@localhost'])
+    subprocess.check_call(['git', 'config', 'user.name', 'gitianuser'])
     subprocess.check_call(['git', 'checkout', '963322de8420c50502c4cc33d4d7c0d84437b576'])
+    subprocess.check_call(['git', 'fetch', 'origin', '72c51f0bd2adec4eedab4dbd06c9229b9c4eb0e3'])
```
Comment (hyc) [inline contrib/gitian/gitian-build.py]: Just to note, this commit has been merged upstream https://github.com/devrandom/gitian-builder/pull/219

Comment (hyc) [reply contrib/gitian/gitian-build.py]: ping @TheCharlatan

Comment (TheCharlatan) [reply contrib/gitian/gitian-build.py]: The reason why we pin a certain commit hash is to ensure that we all use the same gitian scripts for building. I think we should update to latest master (or at least to this merge).

**Comment by dEBRUYNE-1** [comment]
@hyc - would you mind rebasing this PR?

**Pull request merged**